docs/Ways-of-Working/Guides/Engineering/Staff-Engineering/index.md

```diff
@@ -0,0 +1,63 @@
+
+### Technical design support
+
+A staff engineer can use their existing domain or technology expertise to help a delivery team create a technical design for some upcoming work. In most cases, participating in technical design will involve taking part in a design session; this may be anything from an hour to multiple days, depending on the scope and complexity of the work being designed.
+
+Even when they lack technical expertise in a relevant area, participating in technical design sessions (or being updated with the relevant outcomes) can be an effective way for staff engineers to develop and maintain their domain knowledge across areas of our technical estate they are less familiar with.
+
+Design sessions should appear as events in the staff engineer's calendar, which should serve to communicate their availability. However, extended design sessions should be communicated in advance with the rest of the Support Hub, to help mitigate the impact of their absence.
+
+### Delivery team support
+
+A staff engineer will often spend some time embedded within a delivery team. This will often be a natural follow-up to help implement something they have helped in the design phase of. It could also be providing practical support to a team working with a particular technology or part of the domain that the staff engineer has expertise with, or it could simply be a way of providing additional software engineering resource to a team that needs it.
+
+Whilst working with another team, the staff engineer will use whatever mechanisms the team they are joining already uses to manage their work and record their time.
+
+!!! note
+    Given that staff engineers can work across any team, delivery teams should be aware that the staff engineer joining them to provide support may not be very familiar with their ways of working. Estimation is likely to be a particular challenge, as different groups of people will develop different shared subjective scales.
+
+## Working with Staff Engineers
+
+This section is particularly aimed at anyone who wants to get some support from a staff engineer, including:
+
+- Delivery teams working on particularly complex or unknown problems
+- Product Managers working on a problem that hasn't yet been assigned to a delivery team
+
+Their is currently no set process for "booking" an engineer's time, so start by having an open conversation with them to discuss your needs and how they might be able to support. Be mindful that staff engineers will have commitments within the Support Hub, as well as requests from teams or department members, so engage with them early! It may also be worth bringing a Delivery Manager into the conversation, as they should have a good overview of delivery plans across the department.
```

Comment:
use this? There is currently no set process for "booking" an engineer's time, we expect the process will evolve over time, iteratively by gathering feedback, so please do provide feedback whenever possible.

Here is some initial guidance for getting started: 

- Start by having an open conversation with a staff engineer to discuss need's and define an initial support plan
- Engage Early! Be mindful that staff engineers have commitments within Support Hub, as well requests from teams or department members
- Engage a delivery manager to provide an overview of delivery plans across the department